pkg/routing/routing.go

```diff
@@ -43,6 +43,14 @@ func Subnets(routes []*Route) []*net.IPNet {
 	return ns
 }
 
+func SubnetsOverlap(one, other *net.IPNet) bool {
```

Comment:
I see these as not used ?

![image](https://github.com/telepresenceio/telepresence/assets/16439060/16a0fb8a-66dc-428a-8d7d-6d4221230f47)

Reply:
yeah there's probably a few of those, I'll take a look